modules/dashboard/test/DashboardTest.php

```diff
@@ -36,10 +53,165 @@ class DashboardTest extends LorisIntegrationTest
     public function testDashboardPageLoads()
     {
         $this->safeGet($this->url . '/dashboard/');
-
         $welcomeText = $this->webDriver
             ->findElement(WebDriverBy::cssSelector(".welcome"))->getText();
         $this->assertContains("Welcome", $welcomeText);
     }
+
+
+
+     /**
+      * To test that, when loading the Dashboard, click the Views button of
+      * Recruitment, the items "View overall recruitment" and "View site breakdown"
+      * appear
+      * author : Wang Shen
+      *
+      * @return void
+      */
+    public function testDashboardRecruitmentView()
+    {
+        $this->safeGet($this->url . '/dashboard/');
+        $views = $this->webDriver
+            ->findElement(
+                WebDriverBy::Xpath(
+                    "//*[@id='lorisworkspace']/div/di".
+                    "v[1]/div[2]/div[1]/div/div/button"
+                )
+            );
+        $views->click();
+
+        $assertText1 = $this->webDriver
+            ->findElement(
+                WebDriverBy::XPath(
+                    "//*[@id='lorisworkspace']/div/div[1]".
+                    "/div[2]/div[1]/div/div/ul/li[1]/a"
+                )
+            )->getText();
+        $assertText2 = $this->webDriver
+            ->findElement(
+                WebDriverBy::XPath(
+                    "//*[@id='lorisworkspace']/div/div[1]".
+                    "/div[2]/div[1]/div/div/ul/li[2]/a"
+                )
+            )->getText();
+        $this->assertContains("View overall recruitment", $assertText1);
+        $this->assertContains("View site breakdown", $assertText2);
+    }
+    /**
+  * Verify that for a user with 'conflict_resolver' permission,
+  * Check that site displayed is always 'All'.
+  * Click on this task and verify that you go to the conflict_resolver page.
+  *
+  *@return void
+  */
+    public function testConflictResolverPermission()
+    {
+        // check the element which shows on the My tasks panel
+         $this->setupPermissions(array("conflict_resolver"));
+             // check the link
+             $this->safeGet($this->url . "/conflict_resolver/");
+             $bodyText = $this->safeFindElement(WebDriverBy::cssSelector("body"))
+                 ->getText();
+
+             $this->assertNotContains(
+                 "You do not have access to this page",
+                 $bodyText
+             );
+             $this->resetPermissions();
+
```

Comment:
This code doesn't seem to do what the comment says. It's directly accessing the conflict resolver page, not verifying the site or clicking the link.